src/components/supportServices/query.ts

```diff
@@ -20,3 +21,26 @@ export const formatter: QueryFormatter<NodeSupportService, Contact> = (
     href: entity.field_link.uri,
   }
 }
+
+// takes a Support Service and returns it formatted for benefits hub
+export const supportServiceFormatter: QueryFormatter<
+  NodeSupportService,
+  SupportService
+> = (entity: NodeSupportService) => {
+  if (!entity) return null
+
+  return {
+    type: entity.type,
+    id: entity.id,
+    title: entity?.title || '',
+    number: entity?.field_phone_number,
```

Comment:
@chriskim2311 trying to do a QA of this but the tugboat is failing to build this page `/decision-reviews` I couldn't look locally to give you a hint of what the issue is because my local ddev is rebuilding. But you should be able to get some helpful error messages trying to load that page locally. I imagine that it is happening because this should be set to null if the value is not there.